Integrations/Cybereason/Cybereason.py

```diff
@@ -1029,10 +1027,22 @@ def query_file_command():
             if 'companyName' in simple_values:
                 company_name = simple_values['companyName']['values'][0]
 
+            created_time = None
+            if 'createdTime' in simple_values:
+                created_time = timestamp_to_datestring(simple_values['createdTime']['values'][0])
+
+            modified_time = None
+            if 'modifiedTime' in simple_values:
+                modified_time = timestamp_to_datestring(simple_values['modifiedTime']['values'][0])
+
+            is_signed = None
+            if 'isSigned' in simple_values:
+                is_signed = simple_values['isSigned']['values'][0]
```

Comment:
you changed the value is `is_signed` here, as the API returns a string, and we return a boolean